lib/instances/ScreenGui_spec.lua

```diff
@@ -1,27 +1,32 @@
-local Instance = import("../Instance")
+local ScreenGui = import("./ScreenGui")
 local typeof = import("../functions/typeof")
 
 describe("instances.ScreenGui", function()
 	it("should instantiate", function()
-		local instance = Instance.new("ScreenGui")
+		local instance = ScreenGui:new()
```

Comment:
I think this merge went the wrong way -- for objects that are creatable with `Instance.new` (`ScreenGui` is), we should use `Instance.new` in the unit tests for that object. Otherwise, we have no tests that validate that they stay accessible correctly.

Reply:
oooops got it